htdocs/default/robots.txt

```diff
@@ -1,4 +1,5 @@
 Sitemap: https://www.europeana.eu/sitemap-record-index.xml
+Sitemap: https://www.europeana.eu/sitemap-entity-index.xml
```

Comment:
Does it make sense to have this in the robots.txt for www.europeana.eu?

I am not sure search engines will like the sitemap files containing URLs for a different FQDN.